github/event_types.go

```diff
@@ -190,6 +190,38 @@ type IssueCommentEvent struct {
 	Sender  *User       `json:"sender,omitempty"`
 }
 
+// Triggered when a repository's label is created, edited, or deleted.
```

Comment:
This is a comment documenting `LabelEvent`, it should begin with `// LabelEvent ...`. So:

```Go
// LabelEvent is triggered when ...
```

> The first sentence should be a one-sentence summary that starts with the name being declared.

Source: https://golang.org/doc/effective_go.html#commentary